model/update.go

```diff
@@ -18,7 +18,7 @@ import "time"
 
 // Type info structure
 type ArtifactUpdateTypeInfo struct {
-	Type string `json:"type" valid:"required"`
+	Type *string `json:"type" valid:"required"`
```

Comment:
I experimented a bit with this. It is "required" in the sense that it will always be there when parsing the Mender Artifact. Is only that its value is `nil`. That is why I ended up leaving it.